src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html

```diff
@@ -20,13 +21,80 @@
         ></app-text-input>
       </mat-card>
     </div>
+  </div>
+
+  <div class="row" *ngIf="algorithm.computationModel === 'QUANTUM' && frontendAlgorithm.computationModel === 'QUANTUM'">
+    <!-- TODO: finish Sketch -->
```

Comment:
What's with this TODO?

Reply:
(it is an old comment) I think the idea was to add a sketch component (e.g. for showing circuits, as in the platform) supporting the upload of images.

Reply:
If this is not realized yet we should add it as an issue to track this todo.

Reply:
Here it is https://github.com/UST-QuAntiL/qc-atlas-ui/issues/164